### PR TITLE
A function to return a boolean if word is palindrome

### PR DESCRIPTION
**Pull request by Dove-white**
## Summary
# A function to return true or false if the word is a palindrome.

# Exercise [Link](https://docs.google.com/document/d/1AbAe_OAi_LwAgBUZN7N2k-0E7VS6JfLmnT08r16PdBE/edit)

### CODE REVIEW

File: palindrome.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  for (let value of arguments) {
+    let splitValue = value.split("");
+    let revereSplitValue = splitValue.reverse();
+    let removeSplit = revereSplitValue.join("");
+
+    return removeSplit === value;
+  }
+}
+
+console.log(isPalindrome(word));
```
Comment (SekoViper) [inline palindrome.js]: Hello @Dove-white, 

- The variable name "word" is descriptive and appropriate. However, it's worth noting that the code does not only considers a single word, so the variable name could be more specific, such as "inputValue" or "givenValue". 
- The code defines a function named isPalindrome() that appears to expect multiple arguments. The questions is expecting you check a string therefore, your function can be defined with an argument (one argument) to the function to make it specific and easy to work with.
- Your code does run well but does not pass some test cases. Try to run your code with other edge cases before submitting. (e.g "Sit on a potato pan, Otis", "Cigar? Toss it in a can. It is so tragic", "Taco cat", "Mom", "Nun" and many others are Palindromes.)

**Review by SekoViper — CHANGES_REQUESTED** [review]
# Status: Require Changes :no_entry: 
Hello @Dove-white, 
Please check the comments below to improve your code.
